akka-remote/src/main/scala/akka/remote/serialization/MessageContainerSerializer.scala

```diff
@@ -31,6 +26,8 @@ class MessageContainerSerializer(val system: ExtendedActorSystem) extends BaseSe
   def includeManifest: Boolean = false
 
   def toBinary(obj: AnyRef): Array[Byte] = obj match {
+    case identify: Identify         ⇒ serializeIdentify(identify)
+    case identity: ActorIdentity    ⇒ serializeActorIdentity(identity)
```

Comment:
place those after `ActorSelectionMessage` since the selection is probably used more often